NEWcafelatte/cafelatte.py

```diff
@@ -14,3 +15,5 @@ def prepare_coffee_time(self):
         time_to_prepare = 4 * self.coffee_spoons + 2 * self.sugar_spoons + self.milk/100
```

Comment:
pep8 recommends to leave spaces between operators.
https://www.python.org/dev/peps/pep-0008/#other-recommendations

Reply:
done